src/webgpu/util/conversion.ts

```diff
@@ -382,7 +382,8 @@ export class Scalar {
 
 /** Create an f32 from a numeric value, a JS `number`. */
 export function f32(value: number): Scalar {
-  return new Scalar(TypeF32, value, new Float32Array([value]));
+  const arr = new Float32Array([value]);
```

Comment:
We'll want to do this for `i32`, `u32`, `bool` too.

Reply:
Done